### PR TITLE
version control updates

### PR DESCRIPTION
**Pull request by lindsaycarr**
Incorporating feedback from Google doc, except SSH instructions are in #209

### CODE REVIEW

File: content/r-package-dev/Version_Control.Rmd [hunk 1]
```diff
@@ -13,7 +13,7 @@ weight: 1
 library(knitr)
 
 knit_hooks$set(plot=function(x, options) {
-  sprintf("<img src='../%s%s-%d.%s'/ title='%s'/>", 
+  sprintf("><img src='../%s%s-%d.%s'/ title='%s'/>",
```
Comment (jiwalker-usgs) [inline content/r-package-dev/Version_Control.Rmd]: Did this `>` sneak in accidentally?

File: content/r-package-dev/Version_Control.Rmd [hunk 2]
```diff
@@ -85,73 +85,87 @@ There are many ways to use Git, GitHub, and RStudio in your version control work
...
+In the command prompt, type `git remote -v` and hit enter. This will show you which remote repositories (available on online) are connected to your local repository. You should initially only see your forked repository and it is labeled "origin". To add the main repo as an "upstream" repository, type `git remote add upstream <SSH key>` with the correct SSH key and hit enter. E.g. `git remote add upstream git@github.com:USGS-R/dataRetrieval.git`. Now when you run `git remote -v` in the shell, you should see both origin and upstream listed. This means you are set up to get new changes from the main repo and add your local changes to your remote repo. We will see how to do that next.
```
Comment (jiwalker-usgs) [inline content/r-package-dev/Version_Control.Rmd]: I think ssh key is the wrong term here.  Maybe ssh URL instead?

File: content/r-package-dev/Version_Control.Rmd [hunk 3]
```diff
@@ -186,11 +214,11 @@ If you'd prefer the latter method, follow the blog until the "Merging branches b
...
 4. Now follow the rest of steps for completing your PR submission as described in the [how-to-submit-a-PR section](#submitting-pr).
 
-![](../static/img/pr_change_branch.png)
+![Dropdown menu to change branches for PR](../static/img/pr_change_branch.png#inline-img "branch PR")
 
 ## gitignore file
 
-It is sometimes useful to have a text file name `.gitignore`. This file let's Git know which files it should not worry about tracking. For RStudio projects, it's a good idea to have the `.Rproj` and `.Rhistory` files specified in a gitignore. You can use `*` before a file extension to say any file with that extension should be ignored. Here's an example of what the `.gitignore` content might look like:
+It is sometimes useful to have a text file name `.gitignore`. This file let's Git know which files it should not worry about tracking. For RStudio projects, it's a good idea to have the `.Rproj` and `.Rhistory` files specified in a gitignore. You can use `*` before a file extension to say any file with that extension should be ignored (including ones in sub-folders). Here's an example of what the `.gitignore` content might look like:
```
Comment (jiwalker-usgs) [inline content/r-package-dev/Version_Control.Rmd]: perhaps "ones" -> "those"

File: content/r-package-dev/Version_Control.Rmd (hunk 1 above)
Comment (lindsaycarr) [inline content/r-package-dev/Version_Control.Rmd]: Yeah, I think so!

Comment (lindsaycarr) [inline content/r-package-dev/Version_Control.Rmd]: I think it got caught up in a find and replace all

File: content/r-package-dev/Version_Control.Rmd (hunk 2 above)
Comment (lindsaycarr) [inline content/r-package-dev/Version_Control.Rmd]: hmmm OK. Reasons why? You might want to add those comments to #209 as well :)

Comment (wdwatkins) [inline content/r-package-dev/Version_Control.Rmd]: Yeah, the key is what you set up on your local machine for authentication

Comment (wdwatkins) [inline content/r-package-dev/Version_Control.Rmd]: Or ssh address?

Comment (lindsaycarr) [inline content/r-package-dev/Version_Control.Rmd]: Hmm address does seem better.

**Comment by wdwatkins** [comment]
@lindsaycarr I think [here](https://github.com/USGS-R/training-curriculum/pull/210/files#diff-2f1b1be4c5cb66724971e0f281919076L60) you mean `origin` instead of master?  Master would just be the branch you are using.

**Comment by wdwatkins** [comment]
Also I think this https://github.com/USGS-R/training-curriculum/pull/210/files#diff-2f1b1be4c5cb66724971e0f281919076R184 is backwards - the top chunk (HEAD) is what you have locally, the bottom is what you tried to merge in.

**Comment by lindsaycarr** [comment]
The links aren't actually linking to any specific line, so I'm not sure which origin vs master you are referring to. I think I found it though.

You are definitely correct on the merge conflict point. The image I have is correct, but the text is not....thanks for catching that!

Check out this next commit and tell me if that fixed what you were referring to.

**Comment by wdwatkins** [comment]
Yeah, looks like you got the other places too 👍 

One other thing that might be a bit nitpicky, but I would maybe use "message" instead of "comment" for the commit section.  Message is I think the term git uses (the terminal command is `git commit -m` to add a message), and it avoids confusion with code comments.

**Comment by lindsaycarr** [comment]
I'll add that then merge this! Any other super nitpicky items can come after the actual workshop :)

**Comment by lindsaycarr** [comment]
This is the rest of #153

**Pull request merged**